runner/flag_test.go

```diff
@@ -103,9 +103,9 @@ func TestConfigRuntimeArgs(t *testing.T) {
 		},
 		{
 			name: "check exclude_regex",
```

Comment:
Hmm what would you expect in that case, the default values ore empty slice? It seems currently it keeps the default values (I didn't think of this case before), but I'd rather expect to set it to empty.

Reply:
I prefer set it to empty

Reply:
Unfortunately I think it isn't that straightforward. Before a value is set there's a check for nil and empty string https://github.com/cosmtrek/air/blob/master/runner/config.go#L324. If I only keep the nil check it would work (with an additional change in `util.go` to explicitly set an empty slice), but that will override all the default values because arguments that are not explicitly set seem to end up as empty strings instead of `nil`. Do you think it would be easy to adjust that non-set arguments would be `nil` instead of empty strings? I don't know the code base well enough yet and I have not so much experience with the `flag` package.

Reply:
If I see correctly it's only possible to define string variable flags with a string as default value (not a pointer, thus impossible to set `nil` as default value).

See also here https://github.com/cosmtrek/air/blob/master/runner/flag.go#L12.

Reply:
we can set default string as unset valuehttps://github.com/cosmtrek/air/pull/300/files#diff-19e19f16fa5018b7ad46fc960cdbe0138b0029dac020b42c798a9ff9b8203493R7 which is user use to set.

then filter the unset value in there https://github.com/cosmtrek/air/pull/300/files#diff-0a9b65ef9e6a4d3b9e2aec07d2dbaed8bb3866d74d3a2c78d78ddf56ed9aa071L324 

u can cherry-pick my commit

Reply:
Thanks for your support. I cherry-picked all 3 commits.